utils/getScenarioContent.tsx

```diff
@@ -3,14 +3,12 @@ import { Claim, ClaimDetailsContent, ClaimStatusContent, ScenarioContent } from
 export enum ScenarioType {
   PendingDetermination = 'Pending determination scenario',
   BasePending = 'Base state with pending weeks',
-  BaseNoPending = 'Base state with no pending weeks',
+  BaseNoPendingActive = 'Base state with no pending weeks; Active claim',
+  BaseNoPendingInactive = 'Base state with no pending weeks; Inactive claim',
```

Comment:
I find myself really torn on staying brief and wanting more clarity. 

Pending is overloaded and applies to both Determinations and Weeks, so should we always include the associated Noun?

Active/Inactive apply the a whole other object (the claim), should we include that?

But then we have names like:
```
  BaseNoPendingWeeksActiveClaim 
  BaseNoPendingWeeksInactiveClaim 
```
is that just getting absurd and hard to parse?

Reply:
Totally hear you. I'll take another pass at this.